src/templates/testribbon.html

```diff
@@ -0,0 +1,11 @@
+<div class="corner-ribbon">
+  <div class="wrapper">
```

Comment:
This element seems unnecessary according to your scss?

Reply:
good catch, legacy from before i factorised